deploy/lvm-operator.yaml

```diff
@@ -95,6 +95,12 @@ spec:
                 description: Capacity of the volume
                 minLength: 1
                 type: string
+              formatOptions:
+                description: FormatOptions specifies additional options while formatting
```

Comment:
Is this upgrade safe? I mean if you upgrade CRD when there are old version of LVMVolume cr present on cluster?

Reply:
It's just a new optional field, nothing will happen...

Reply:
I'd say.  Lets still test it to be sure.

Reply:
Okay :), will test it out after writing the unit tests

Reply:
It was fine in my tests...